structs.go

```diff
@@ -1395,38 +1395,44 @@ type Intent int
 
 // Constants for the different bit offsets of intents
 const (
-	IntentsGuilds                 Intent = 1 << 0
-	IntentsGuildMembers           Intent = 1 << 1
-	IntentsGuildBans              Intent = 1 << 2
-	IntentsGuildEmojis            Intent = 1 << 3
-	IntentsGuildIntegrations      Intent = 1 << 4
-	IntentsGuildWebhooks          Intent = 1 << 5
-	IntentsGuildInvites           Intent = 1 << 6
-	IntentsGuildVoiceStates       Intent = 1 << 7
-	IntentsGuildPresences         Intent = 1 << 8
-	IntentsGuildMessages          Intent = 1 << 9
-	IntentsGuildMessageReactions  Intent = 1 << 10
-	IntentsGuildMessageTyping     Intent = 1 << 11
-	IntentsDirectMessages         Intent = 1 << 12
-	IntentsDirectMessageReactions Intent = 1 << 13
-	IntentsDirectMessageTyping    Intent = 1 << 14
-
-	IntentsAllWithoutPrivileged = IntentsGuilds |
-		IntentsGuildBans |
-		IntentsGuildEmojis |
-		IntentsGuildIntegrations |
-		IntentsGuildWebhooks |
-		IntentsGuildInvites |
-		IntentsGuildVoiceStates |
-		IntentsGuildMessages |
-		IntentsGuildMessageReactions |
-		IntentsGuildMessageTyping |
-		IntentsDirectMessages |
-		IntentsDirectMessageReactions |
-		IntentsDirectMessageTyping
+	IntentGuilds                 Intent = 1 << 0
+	IntentGuildMembers           Intent = 1 << 1
+	IntentGuildBans              Intent = 1 << 2
+	IntentGuildEmojis            Intent = 1 << 3
+	IntentGuildIntegrations      Intent = 1 << 4
+	IntentGuildWebhooks          Intent = 1 << 5
+	IntentGuildInvites           Intent = 1 << 6
+	IntentGuildVoiceStates       Intent = 1 << 7
+	IntentGuildPresences         Intent = 1 << 8
+	IntentGuildMessages          Intent = 1 << 9
+	IntentGuildMessageReactions  Intent = 1 << 10
+	IntentGuildMessageTyping     Intent = 1 << 11
+	IntentDirectMessages         Intent = 1 << 12
+	IntentDirectMessageReactions Intent = 1 << 13
+	IntentDirectMessageTyping    Intent = 1 << 14
+	IntentMessageContent         Intent = 1 << 15
+	IntentGuildScheduledEvents   Intent = 1 << 16
+
+	IntentsAllWithoutPrivileged = IntentGuilds |
+		IntentGuildBans |
+		IntentGuildEmojis |
+		IntentGuildIntegrations |
+		IntentGuildWebhooks |
+		IntentGuildInvites |
+		IntentGuildVoiceStates |
+		IntentGuildMessages |
+		IntentGuildMessageReactions |
+		IntentGuildMessageTyping |
+		IntentDirectMessages |
+		IntentDirectMessageReactions |
+		IntentDirectMessageTyping |
+		IntentGuildScheduledEvents
+
 	IntentsAll = IntentsAllWithoutPrivileged |
```

Comment:
Not sure if that's a good idea to put message content intent into unpriveleged intents. In the future it would be a priveleged intent.